docs/ops/arithmetic/Acosh_3.md

```diff
@@ -6,32 +6,28 @@
 
 **Short description**: *Acosh* performs element-wise hyperbolic inverse cosine (arccosh) operation with given tensor.
 
-**Attributes**:
+**Detailed description**:  Operation takes one input tensor and performs the element-wise hyperbolic inverse cosine operation on a given input tensor, based on the following mathematical formula:
 
-    No attributes available.
+\f[
+a_{i} = acosh(a_{i})
+\f]
+
+**Attributes**: *Acosh* operation has no attributes.
 
 **Inputs**
 
-* **1**: A tensor of type *T*. **Required.**
+* **1**: A tensor of type *T* and arbitrary shape. **Required.**
 
 **Outputs**
 
-* **1**: The result of element-wise acosh operation. A tensor of type *T*.
+* **1**: The result of element-wise *Acosh* operation. A tensor of type *T* and the same shape as the input tensor.
 
 **Types**
 
-* *T*: any floating-point type.
-
-*Acosh* does the following with the input tensor *a*:
-
-\f[
-a_{i} = acosh(a_{i})
-\f]
+* *T*: any numeric type.
```

Comment:
do we support integer type? I see there is no framework that support integer type.

Reply:
In ngraph we do: https://github.com/openvinotoolkit/openvino/blob/master/ngraph/core/src/op/acosh.cpp#L42-L53.
Also CPU tests have same as ref impl output for acosh.